frontend/src/core/user/reducer.js

```diff
@@ -55,7 +55,7 @@ export type UserState = {|
     +username: string,
     +managerForLocales: Array<string>,
     +translatorForLocales: Array<string>,
-    +translatorForProjects: Array<string>,
+    +translatorForProjects: Object,
```

Comment:
Can you be more precise in this declaration? Based on what you told me over IRC, I think this should be `{ [string]: boolean }` (see documentation: https://flow.org/en/docs/types/objects/#toc-objects-as-maps ).